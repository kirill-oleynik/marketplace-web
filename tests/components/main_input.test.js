const React = require('react');
const renderer = require('react-test-renderer');
const MainInput = require('../../app/components/main_input').default;

describe('#render', () => {
  test('it renders correctly', () => {
    const tree = renderer.create(
      <MainInput
        name="test"
        label="Test"
      />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
