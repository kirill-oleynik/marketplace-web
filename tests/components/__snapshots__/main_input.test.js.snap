// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<div
  className="main-input"
>
  <div
    className="main-input__field-box"
  >
    <input
      className="main-input__field"
      name="test"
      type="text"
      value=""
    />
    <label
      className="main-input__label"
      htmlFor="test"
    >
      Test
    </label>
  </div>
</div>
`;
